lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -471,6 +472,8 @@ namespace Js
 
         mutable Field(CharStringCache) charStringCache;
 
+        Field(ScriptContextPolymorphicInlineCache*) toStringTagCache;
```

Comment:
Can you add setting this to nullptr in the initializer list for the constructor?

Reply:
nit: there is a `Cache` struct up above, might be better to include this in there

---
In reply to: [144174060](https://github.com/Microsoft/ChakraCore/pull/3936#discussion_r144174060) [](ancestors = 144174060)

Reply:
Done; will update PR shortly.